src/flag_gems/runtime/backend/backend_utils.py

```diff
@@ -13,10 +13,20 @@ class VendorInfoBase:
 
 
 def get_tune_config(vendor_name, file_mode="r"):
-    vendor_name = "_" + vendor_name
-    script_path = os.path.abspath(__file__)
-    file_path = os.path.dirname(script_path) + "/" + vendor_name + "/tune_configs.yaml"
-    with open(file_path, file_mode) as file:
-        config = yaml.safe_load(file)
+    try:
+        if not vendor_name.isalnum():
+            raise ValueError("Invalid vendor name")
```

Comment:
function is called inside runtime code, so vendor_name can hardly be invalid.

Reply:
removed